app/src/main/java/org/oppia/android/app/devoptions/markchapterscompleted/MarkChaptersCompletedFragmentPresenter.kt

```diff
@@ -78,6 +78,14 @@ class MarkChaptersCompletedFragmentPresenter @Inject constructor(
               )
           }
         }
+      } else if (binding.isAllChecked == true) {
+        binding.isAllChecked = false
+        getMarkChaptersCompletedViewModel().getItemList().forEach { viewModel ->
+          if (viewModel is ChapterSummaryViewModel) {
+            if (!viewModel.checkIfChapterIsCompleted())
+              chapterUnselected(viewModel.chapterIndex, viewModel.nextStoryIndex)
```

Comment:
This should have braces per https://developer.android.com/kotlin/style-guide#formatting (we only omit braces if the entire if condition + conditioned statement can fit on one line).

Reply:
Noted updated the statements.